core/lexicon/en/setting.inc.php

```diff
@@ -438,6 +438,9 @@
 $_lang['setting_manager_login_url_alternate'] = 'Alternate Manager Login URL';
 $_lang['setting_manager_login_url_alternate_desc'] = 'An alternate URL to send an unauthenticated user to when they need to login to the manager. The login form there must login the user to the "mgr" context to work.';
 
+$_lang['setting_manager_tooltip_enable'] = 'Manager tooltip enable';
+$_lang['setting_manager_tooltip_delay'] = 'Manager tooltip delay';
```

Comment:
```suggestion
$_lang['setting_manager_tooltip_delay'] = 'Delay Time for Manager Tooltips';
```

Please consider adding a descriptive lexicon `setting_manager_tooltip_delay ` explaining the setting and value.